### ISSUE TITLE
use statement order sort fix

### ISSUE THREAD

**Issue by dereuromark** [cakephp/cakephp-codesniffer#169]
Can we please make stickler and the code sniffer in sync with the normal expectation of ordering?

stickler-ci reports it the other way around, which then must be resolved by hand :(
```
use Cake\Routing\Router;
use Cake\Routing\Route\DashedRoute;
```
See https://github.com/dereuromark/cakephp-tools/commit/7ca32edfab80e2dcf2b207d8c107a9e78314add3

IDEs though provide it by natural sorting of directories:
```
use Cake\Routing\Route\DashedRoute;
use Cake\Routing\Router;
```

Just a minor glitch, though.

**Comment by markstory** [comment]
stickler-ci is using this repo for linting rules. If we change our rules it will cause big churns in related repositories.

**Comment by dereuromark** [comment]
I guess there is no good solution then.

**Comment by ADmad** [comment]
@dereuromark Have you identified exactly which sniff is reporting the error? If it's one inherited from PSR2 ruleset you can submit a ticket on phpcs' repo itself.

**Comment by dereuromark** [comment]
Well, maybe the IDEs are doing it wrong, not sure anymore what the "correct" order would be.
